### DISCUSSION TITLE
React Joyride latest version is not working

### DISCUSSION THREAD

**Discussion by janinebalangue** [gilbarbara/react-joyride#961] [Q&A]
Hi everyone, 

I need some help figuring out this error I have encountered using the react-joyride in our project.

For context, we are using the library in onboarding new users and new features in the application on our ongoing project, and about 2-3weeks ago, we encountered a problem in our deployment pipelines when we were deploying a new version of our ongoing project. Upon thorough debugging/investigation, we found that it was caused by this library for the latest updates of the library has something to do with its dependency packages also. Please see below the screenshot of the errors for reference

_We encountered this error even tho we have a 2.5.3 version installed, we also tried upgrading it to 2.5.5 and 2.6.1 versions but both versions are also displaying the same errors on our deployment pipelines._
![image](https://github.com/gilbarbara/react-joyride/assets/149121230/7203bce8-9871-4fe5-ae6f-539ed537475a)

_We it comes to our local machine, the errors are showing on versions 2.5.5, 2.6.1, and 2.6.2_
![image](https://github.com/gilbarbara/react-joyride/assets/149121230/1da7f873-ee2d-4402-a688-da377b14c86d)

For other references:
![image](https://github.com/gilbarbara/react-joyride/assets/149121230/6acc0d77-c425-47d9-95f9-53a8bd22e70d)

Thank you in advance for your help and assistance

**Comment by gilbarbara** [comment]
Hey @janinebalangue 
What are the versions of react, @types/react, and typescript in the project?

You can pin the 2.5.5 version by removing the `^` character from the version number in your package.json.

**Reply by janinebalangue** [reply]
Hi @gilbarbara,

**_What are the versions of react, @types/react, and typescript in the project?_**
- **React**: 17.0.2
- **@types/react**: not installed
- **typescript**: 5.0.2

**_You can pin the 2.5.5 version by removing the ^ character from the version number in your package.json_** - also tried this one but it just gave us the same error.

Thank you.

**Reply by gilbarbara** [reply]
react-joyride before 2.6.x exports from `es/index.js`, so if you still see `dist/index.mjs`, you are running the latest version.

Make sure your **package.json** looks like this:
```
{
	"dependencies": {
		...
		"react-joyride": "2.5.5"
	}
}
And run `npm install`
